examples/bri-3/.env sample

```diff
@@ -4,4 +4,4 @@
 # Prisma supports the native connection string format for PostgreSQL, MySQL, SQLite, SQL Server, MongoDB and CockroachDB.
 # See the documentation for all the connection string options: https://pris.ly/d/connection-strings
 
-DATABASE_URL="file:./dev.db"
```

Comment:
this file should be named `.env.sample` not `.env sample`

please add separate `.env` file and git ignore it, `.env.sample` is just used to show which variables need to be set for local development and `.env` will be set up by everyone locally

also usual postgres port is 5432

Reply:
5433 is to prevent a local db already running on that port from interfering.

Reply:
I have my own .env which is ignored. I added to the sample file so that other devs have an example string for spinning up docker in their own .env file.

Reply:
Okay, I can make that change. I did not create this sample file, only edited it.